DEVELOPMENT.md

```diff
@@ -166,15 +166,118 @@ The actual commands for compilation are handled by our Makefile, so the only thi
 ├── pkg                 contains all libraries used in the TTN stack for LoRaWAN
 │   ├── component       contains the base component; all other components extend this component
 │   ├── config          package for configuration using config files, environment and CLI flags
+│   ├── console         package that provides the web server for the console
 │   ├── errors          package for rich errors that include metadata and cross API boundaries
 │   ├── log             package for logging
 │   ├── messages        contains non-proto messages (such as the messages that are sent over MQTT)
 │   ├── metrics         package for metrics collection
 │   ├── ttnpb           contains generated code from our protocol buffer definitions and some helper functions
 │   ├── types           contains primitive types
+│   ├── webui           contains js code for the console and oauth provider
 │   └── ...
 ├── public              frontend code will be compiled to this folder - not added to git
 ├── release             binaries will be compiled to this folder - not added to git
 └── sdk                 source code for our SDKs
     └── js              source code for our JavaScript SDK
 ```
+
+
+## Frontend
+### Introduction
+The TTN stack for LoRaWAN employs two frontend applications: the **Console** and **OAuth Provider**. Both applications use [React](https://reactjs.org/) as frontend framework. The `console` package of the backend exposes the webserver and – together with the `oauth` package – is in charge of handling authorization logic (OAuth flow, login/logout).
```

Comment:
`The TTN stack` -> `The Things Network Stack`, right, @johanstokking? We do this wrong in more places in this document.

Reply:
Ah yeah, I was actually writing it this way, but then changed it after I saw it was written differently in the rest of the file.

Reply:
My suggestion would also be to put the brand and stack name always in italics in our docs, makes it easier to scan and less clashing with other parts of the sentence: notice `The The` in `The The Things Network Stack employs …`.

Reply:
There should not be a double "The", it's "The Things Network Stack" or "the stack".